pkg/scheduler/internal/cache/node_tree.go

```diff
@@ -90,7 +90,7 @@ func (nt *NodeTree) addNode(n *v1.Node) {
 		nt.zones = append(nt.zones, zone)
 		nt.tree[zone] = &nodeArray{nodes: []string{n.Name}, lastIndex: 0}
 	}
-	klog.V(5).Infof("Added node %v in group %v to NodeTree", n.Name, zone)
+	klog.V(2).Infof("Added node %v in group %v to NodeTree", n.Name, zone)
```

Comment:
can you use %q instead for the node name? similarly in the other logs for node names and pods.

Reply:
sure thing.